packages/ra-core/src/core/useScrollToTop.tsx

```diff
@@ -23,18 +23,10 @@ import { useHistory } from 'react-router-dom';
  * );
  */
 export const useScrollToTop = () => {
-    const history = useHistory<{ _scrollToTop?: boolean }>();
-    useEffect(
-        () =>
-            history.listen((location, action) => {
-                if (
-                    action !== 'POP' &&
```

Comment:
This was done to preserve scroll on back button, why did you remove it?

Reply:
There's no way to retrieve the current history anymore

Reply:
Then when I'm going back to a list from a show view, I don't return to the scroll position I was before? This is a big breaking change, we absolutely need that feature.